src/core/layertreemodel.cpp

```diff
@@ -999,3 +1042,65 @@ void FlatLayerTreeModelBase::setLayerInTracking( QgsLayerTreeLayer *nodeLayer, b
 
   emit dataChanged( index, index, QVector<int>() << FlatLayerTreeModel::InTracking );
 }
+
+QgsRectangle FlatLayerTreeModelBase::nodeExtent( const QModelIndex &index, QgsQuickMapSettings *quickMapSettings, const float buffer )
```

Comment:
To be consistent with other parts of the code, I'd rename the parameter to mapSettings.

I know this results in mapSettings.mapSettings() , but that's what we have everywhere else and I like consistency. Also, we are a child of QGIS, so we can see that as our geometry.geometry() equivalent in homage to those good old days ;)

Reply:
renamed :)